events.go

```diff
@@ -137,6 +137,22 @@ type Event interface {
 // concrete event types.
 type BasicEvent struct{ *js.Object }
 
+type EventOptions struct {
+	Bubbles    bool
+	Cancelable bool
+}
+
+func CreateEvent(typeArg string, opts *EventOptions) *BasicEvent {
```

Comment:
Do we want *EventOptions or EventOptions? *EventOptions saves some characters when using the defaults, but EventOptions means we don't need the nil check. Geolocation uses PositionOptions, but there isn't an implementation of Geolocation yet (in this package, anyway) so we could possibly change it. The two should be the same, though.

@shurcooL, what's your opinion?